po/pt_BR.po

```diff
@@ -8025,9 +8025,8 @@ msgid ""
 msgstr ""
 
 #: gnucash/gnome-utils/dialog-transfer.c:1325
-#, fuzzy
 msgid "Finance::Quote must be installed to enable this button."
-msgstr "Aviso: Financeiro::Quotação não está instalada corretamente"
+msgstr "Aviso: Financeiro::Quotação não está instalada corretamente."
```

Comment:
Finance::Quote is the name of a perl program/package. Please do not  translate it!